robot-core/src/test/java/org/obolibrary/robot/MeasureOperationTest.java

```diff
@@ -47,7 +48,7 @@ public void testExtendedMetrics() throws IOException {
     MeasureResult result =
         MeasureOperation.getMetrics(
             ontology, "extended", new CURIEShortFormProvider(new IOHelper().getPrefixes()));
-    assertEquals("SI(D)", result.getSimpleMetricValue(MetricsLabels.EXPRESSIVITY));
+    assertEquals("RRESTRUCINTE+I(D)", result.getSimpleMetricValue(MetricsLabels.EXPRESSIVITY));
```

Comment:
😱

Reply:
:)  [Construct.java](https://github.com/owlcs/owlapi/blob/owlapi-parent-4.5.22/api/src/main/java/org/semanticweb/owlapi/util/Construct.java)
ROLE_DOMAIN_RANGE + CONCEPT_UNION + CONCEPT_INTERSECTION + FULL_EXISTENTIAL + ROLE_TRANSITIVE + Use of data properties